src/chaitanyaKDec19/HomeWork18Jan2020/HomeWork.java

```diff
@@ -0,0 +1,68 @@
+package chaitanyaKDec19.HomeWork18Jan2020;
+
+/*
+ * Write a program to accept string from user and display:
+ * 1. Capital Letters
+ * 2. Digit - Odd number sum
+ * 3. Special Characters
+ * 4. Even Number sum
+ * 5. Small Characters
+ */
+
+public class HomeWork {
+
+	static void displayString(String string) {
```

Comment:
It is always advisable not to have same variable as datatype. You should have variable name may be as inputString.

Reply:
Sure I will keep this in mind going forward 👍👍